storage/overlays/aks/storage-class.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: onepanel
+provisioner: kubernetes.io/azure-disk
+parameters:
+  storageaccounttype: Premium_LRS
```

Comment:
Should this be empty given this statement: "Premium VM can attach both Standard_LRS and Premium_LRS disks, while Standard VM can only attach Standard_LRS disks."?

Reply:
I don't follow, what should not be empty?

Reply:
I'm assuming if we remove `storageaccounttype`, then we'll be able to attach this volume to any VM based on that statement. Not 100% sure though.

Reply:
I pulled this from https://kubernetes.io/docs/concepts/storage/storage-classes/#new-azure-disk-storage-class-starting-from-v1-7-2

Reply:
Let me rephrase, if we remove `storageaccounttype`, then this volume can attach to either VM types. If we make it a `Premium_LRS` disk, then it can only attach to Premium VMs.

Reply:
Ahhhhh okay. Yes, will remove.